ocrd_tesserocr/recognize.py

```diff
@@ -357,35 +383,147 @@ def _process_glyphs_in_word(self, result_it, word, word_xywh):
                 glyph_no += 1
                 result_it.Next(RIL.SYMBOL)
 
+def page_element_unicode0(element):
+    if element.get_TextEquiv():
+        return element.get_TextEquiv()[0].Unicode
+    else:
+        return ''
+
+def page_element_conf0(element):
+    if element.get_TextEquiv():
+        # generateDS does not convert simpleType for attributes (yet?)
```

Comment:
This is a regression, they should be converted, I've reopened https://github.com/OCR-D/core/issues/269

Reply:
But it's like this ever since 1.0.1 – the last working previous version (float conversion) was 1.0.b8.

Moreover, IIRC generateDS documentation says explicitly this is a _won't fix_...

Are you certain this breaks round-trip?

Reply:
> Are you certain this breaks round-trip?

Not necessarily but what's the point of having types in XSD if you don't validate data and don't produce native types when code generating. It's not high in the priority list though. But when I re-generate the next time (there's also the issue with building with parent element refs etc.) I'll have a look.

Reply:
I am with you on this, I was puzzled about this decision myself. But maybe it's just a matter of resources and will change in the future. (Also, remember this only affects atomic attribute types.)